docs/release_process.md

```diff
@@ -0,0 +1,323 @@
+        - [ ] Code changes for release are merged into `develop`
+        - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+## 2. Preliminary Testing
+### Description
+After the release branches are created, preliminary testing should be conducted to ensure that the code is stable and ready for release. This includes running unit tests, integration tests, and any other relevant tests to ensure that the code is functioning as expected.
+
+### Acceptance Criteria
+    - [ ] jpo-ode
+        - [ ] code compiles
+        - [ ] unit tests pass
+        - [ ] program starts up correctly
+        - [ ] http endpoint is reachable
+        - [ ] tims can be successfully pushed to http endpoint
+        - [ ] capable of ingesting messages via udp (see scripts in `scripts/tests` directory)
+            - [ ] tims
+            - [ ] bsms
+            - [ ] ssms
+            - [ ] srms
+            - [ ] spats
+            - [ ] maps
+            - [ ] psms
+    - [ ] asn1_codec
+        - [ ] code compiles
+        - [ ] unit tests pass
+        - [ ] program starts up correctly
+        - [ ] program can be configured for decoding successfully
+        - [ ] program can be configured for encoding successfully
+        - [ ] messages get decoded as expected
+        - [ ] messages get encoded as expected
+    - [ ] jpo-cvdp
+        - [ ] code compiles
+        - [ ] unit tests pass
+        - [ ] program starts up correctly
+        - [ ] messages get consumed as expected
+        - [ ] BSMs inside geofence are retained
+        - [ ] BSMs with a partII section are retained
+        - [ ] BSMs outside geofence are suppressed
+        - [ ] BSMs above speed range are suppressed
+        - [ ] BSMs below speed range are suppressed
+    - [ ] jpo-security-svcs
+        - [ ] code compiles
+        - [ ] program starts up correctly
+        - [ ] program can be successfully configured
+        - [ ] messages can be successfully signed
+    - [ ] jpo-sdw-depositor
+        - [ ] code compiles
+        - [ ] unit tests pass
+        - [ ] program starts up correctly
+        - [ ] messages are consumed successfully
+        - [ ] messages are submitted to the SDX successfully
+    - [ ] jpo-s3-deposit
+        - [ ] code compiles
+        - [ ] program starts up correctly
+        - [ ] deposits can be made to one of the destinations successfully
+    - [ ] jpo-geojsonconverter
+        - [ ] code compiles
+        - [ ] unit tests pass
+        - [ ] program starts up correctly
+        - [ ] program can be configured successfully
+        - [ ] MAP & SPaT messages are consumed successfully
+        - [ ] valid ProcessedMaps & ProcessedSpats are outputted
+    - [ ] jpo-conflictmonitor
+        - [ ] code compiles
+        - [ ] unit tests pass
+        - [ ] program starts up correctly
+        - [ ] program processes SpAT/MAP/BSM messages and generates events as expected (use test scripts for this)
+            - [ ] Test BSM Events
```

Comment:
Lol this looks pretty good so I am going to be a stickler on formatting. Looks like every requirement line is lowercase unless it is an acronym. We should probably make the word "Test" lowercase too. Or capitalize the first letter of everything else. Your call.

Reply:
I've adjusted the capitalization to be lowercase except for acronyms, thanks for your feedback!